Maestro/subscriptions.json

```diff
@@ -78,9 +78,9 @@
             "-Project='tests\\build.proj'",
             "--",
             "/t:UpdateDependenciesAndSubmitPullRequest",
-            "/p:GitHubUser=dotnet-bot",
-            "/p:GitHubEmail=dotnet-bot@microsoft.com",
-            "/p:GitHubAuthToken=`$(`$Secrets['DotNetBotGitHubPassword'])",
+            "/p:GitHubUser=dotnet-maestro-bot",
+            "/p:GitHubEmail=dotnet-maestro-bot@microsoft.com",
+            "/p:GitHubAuthToken=`$(`$Secrets['DotNetMaestroBotGitHubToken'])",
```

Comment:
I don't see this secret in the executor builds--are you adding it? For an overall strategy I suggest:

 1. Add `DotNetMaestroBotGitHubToken` to the VSTS defs (keeping `DotNetBotGitHubPassword` for now)
 2. Merge this PR
 3. Wait a while to make sure each def still works
 4. Remove `DotNetBotGitHubPassword` from each def